package.json

```diff
@@ -11,31 +11,30 @@
     "vow-fs": "^0.3.4"
   },
   "devDependencies": {
-    "bem-components": "bem/bem-components#v2.1.0",
```

Comment:
It's an arguable point. In my opinion, developers can solve dependencies resolving by ourself.